### DISCUSSION TITLE
CheckBox column not know

### DISCUSSION THREAD

**Discussion by iranandroid** [filamentphp/filament#4902] [Help] [answered]
Hi, i wanna use checkbox column in table for create editable column, but when use CheckBoxCloumn according to this [documentation](https://filamentphp.com/docs/2.x/tables/columns/checkbox), get error

<img width="920" alt="image" src="https://user-images.githubusercontent.com/20900080/201313373-599c07a0-b2eb-4fb3-a285-a28843f6fb0d.png">

is it possible use check box or switch for editing state in column?

**Comment by danharrin** [comment, marked as answer]
Upgrade Filament

**Reply by iranandroid** [reply]
> Upgrade Filament

I used last version of filament
<img width="500" alt="image" src="https://user-images.githubusercontent.com/20900080/201339919-24e3b4d6-698f-4db2-9b1b-b62a86ddbec4.png">

**Reply by zepfietje** [reply]
Might have to restart Visual Studio Code for intelephense to pick up the changes?

**Reply by iranandroid** [reply]
> Might have to restart Visual Studio Code for intelephense to pick up the changes?

Yes, that's right, thank you
